samples/Azure.Management.Storage/readme.md

```diff
@@ -30,5 +30,9 @@ operation-group-to-resource:
 resource-rename:
    BlobServiceProperties: BlobService
    FileServiceProperties: FileService
+resource-to-parent:
```

Comment:
the name implies the key is the name of the resource however FileShares is actually FileShare as we see in the resource rename

is this really intended to be an operation-group-to-parent?